x-pack/plugins/apm/public/components/shared/PropertiesTable/index.tsx

```diff
@@ -55,17 +62,25 @@ export function AgentFeatureTipMessage({ featureName, agentName }) {
   );
 }
 
-export function sortKeysByConfig(object, currentKey) {
+export const sortKeysByConfig: KeySorter = (object, currentKey) => {
   const presorted = _.get(
     indexedPropertyConfig,
     `${currentKey}.presortedKeys`,
     []
   );
   return _.uniq([...presorted, ...Object.keys(object).sort()]);
-}
+};
 
-export function PropertiesTable({ propData, propKey, agentName }) {
-  if (!propData || Object.keys(propData).length === 0) {
+export function PropertiesTable({
+  propData,
+  propKey,
+  agentName
+}: {
+  propData: StringMap<any>;
+  propKey: string;
+  agentName: string;
+}) {
+  if (Object.keys(propData).length === 0) {
```

Comment:
I normally use `_.isEmpty(propData)`. Do you prefer this for performance?

Reply:
I never think about `_.isEmpty` because I honestly always forget what it does exactly. :P I can switch though, no real preference from me.